oll_sc/api.py

```diff
@@ -22,7 +78,11 @@ def sc_is_present(pkcs11=None):
 @contextmanager
 @init_pkcs11
 def sc_session(pin, pkcs11=None):
-  """Try to log in with provided PIN and return session."""
+  """Open token session needed for signing, encryption, etc.
+
+  Arguments:
+    - pin(str): Pin for session login
```

Comment:
I have omitted it for all API functions, because `init_pkcs11` decorator is responsible for setting its value. This parameter is needed for testing purposes. Do you think I should add it to _docstring_  anyway and note that it should not be set?

Reply:
I think that it's important to note that it should not be set